web/react/components/access_history_modal.jsx

```diff
@@ -4,48 +4,50 @@
 var UserStore = require('../stores/user_store.jsx');
 var AsyncClient = require('../utils/async_client.jsx');
 var LoadingScreen = require('./loading_screen.jsx');
-var utils = require('../utils/utils.jsx');
+var Utils = require('../utils/utils.jsx');
 
-function getStateFromStoresForAudits() {
-    return {
-        audits: UserStore.getAudits()
-    };
-}
+export default class AccessHistoryModal extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.onAuditChange = this.onAuditChange.bind(this);
+        this.handleMoreInfo = this.handleMoreInfo.bind(this);
 
-module.exports = React.createClass({
-    displayName: 'AccessHistoryModal',
-    componentDidMount: function() {
-        UserStore.addAuditsChangeListener(this.onListenerChange);
-        $(this.refs.modal.getDOMNode()).on('shown.bs.modal', function() {
+        this.state = this.getStateFromStoresForAudits();
+        this.state.moreInfo = [];
+    }
+    getStateFromStoresForAudits() {
+        return {
+            audits: UserStore.getAudits()
+        };
+    }
+    componentDidMount() {
+        UserStore.addAuditsChangeListener(this.onAuditChange);
+        $(this.refs.modal.getDOMNode()).on('shown.bs.modal', function show() {
             AsyncClient.getAudits();
         });
 
         var self = this;
```

Comment:
The style guide says that we shouldn't save references to this and should instead be binding functions to this.